characterize.py

```diff
@@ -106,7 +108,7 @@ def __init__(self, config=None):
 
     def start(self):
         hachoir_config.quiet = True
-        self.bad_link_re = re.compile("http[s]?://|powershell|cscript|wscript|mshta|<script")
+        self.bad_link_re = BAD_LINK_RE
```

Comment:
You don't need to be creating this class attribute at all, the only place it is used below is read only, it could just be accessing the constant directly there.

Reply:
Addressed [here](https://github.com/CybercentreCanada/assemblyline-service-characterize/pull/1/commits/28f9560718caab322453eda94cb0ff4d7e6b0f9d)